docs/content/how-to/mce/Dual/hostedcluster/nodepool.md

```diff
@@ -0,0 +1,52 @@
+NodePool is a scalable set of worker nodes attached to a HostedCluster. NodePool machine architectures are uniform within a given pool, and are independent of the control plane’s underlying machine architecture.
+
+This is how looks like:
+
+```yaml
+apiVersion: hypershift.openshift.io/v1beta1
+kind: NodePool
+metadata:
+  creationTimestamp: null
+  name: hosted-dual
+  namespace: clusters
+spec:
+  arch: amd64
+  clusterName: hosted-dual
+  management:
+    autoRepair: false
+    upgradeType: InPlace
+  nodeDrainTimeout: 0s
+  platform:
+    type: Agent
+  release:
+    image: registry.hypershiftbm.lab:5000/openshift/release-images:4.14.0-0.nightly-2023-08-29-102237 ## CHANGE THIS!!
+  replicas: 0
+status:
+  replicas: 0
+```
+
+**Details**:
+
+- All the nodes included in this NodePool will be based on Openshift `4.14.0-0.nightly-2023-08-29-102237` version.
+- The Upgrade type is set to `InPlace` which means that the same bare metal node will be re-used on a upgrade situation.
+- Autorepair is set to `false` because the node will not be recreated when this dissapear.
```

Comment:
```suggestion
- Autorepair is set to `false` because the node will not be recreated when this disappears.
```
```suggestion
- Autorepair is set to `false` because the node will not be recreated when this disappears.
```